Package.swift

```diff
@@ -0,0 +1,107 @@
+// swift-tools-version:4.2
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import PackageDescription
+
+let package = Package(
+  name: "swift-format",
+  products: [
+    .executable(name: "swift-format", targets: ["swift-format"]),
+    .library(name: "SwiftFormat", targets: ["SwiftFormat"]),
+  ],
+  dependencies: [
+    .package(url: "https://github.com/apple/swift-package-manager.git", from: "0.4.0"),
+    .package(
+      url: "https://github.com/apple/swift-syntax",
```

Comment:
No, it shouldn't use relative paths, because then it would become unusable as a dependency of other remote packages.

Reply:
Just a note: If we are going to import this from `sourcekit-lsp`, we'll need it to build against ToT branches of swiftpm and swift-syntax and that works better if we use a relative path when building under `build-script` so that we can do cross-repo PR tests etc.  The way we do this in sourcekit-lsp itself is use a URL+branch dependency by default and use a relative path only when a build-script-provided environment variable is set.

But we can figure out these details later.